scripts/teamDataScript.js

```diff
@@ -0,0 +1,617 @@
+}
+
+
+
+
+//Pit stuff
+function loadPit()
+{
+    var season = getCookie('currCheckedSeason');
+    var cname = season + ' pitQuestions';
+    var event = getCurrEvent();
+    var questions = getCookie(cname).split('Ω'); //Ω is separator for questions
+    var currTeam = getCurrTeamNumber();
+
+    showAllPitDataTable();
+    if (currTeam.trim().length == 0)
+    {
+        $('#pitQuestionTable').html('');
+        $('#pitTableConfirmation').text('No team selected.');
+        return;
+    }
+
+    var tableHeader = '<tr> <th><b>Question</b></th> <th><b>Answer</b></th> </tr>';
+    $('#pitQuestionTable').html(tableHeader); //default start of table
+
+    var localStorageObjName = pitAnswerObjName(season,event,currTeam);
+    var team_QA_pairs = JSON.parse(localStorage.getItem(localStorageObjName)); //get QA pairs that may/may not have already been stored
+    var teamsAlreadyStoredBool = false;
+    if (checkForNull(team_QA_pairs))
+    {
+        teamsAlreadyStoredBool = true;
+    }
+
+    for (var i = 0; i < questions.length;i++)
+    {
+        var question = questions[i];
+        if (question.trim().length == 0) return;
+        
+        //loop through already stored local object: if question has an answer already stored then put that in the input box. Else, leave it empty (to fill in)
+        var potentialAnswer = '';
+        if (teamsAlreadyStoredBool == false) //len of loc storage obj == 0, nothing stored yet
+        {
+            potentialAnswer = '';
+        } else //len of loc storage obj = 1, something stored inside, now find answer to curr question
+        {
+            for (var j = 0; j < team_QA_pairs.length; j++)
+            {
+                var pair = team_QA_pairs[j];
+                var answer = pair[question];
+                if (!checkForNull(answer))
+                {
+                    continue;
+                }
+                potentialAnswer = answer;
+            }
+        }
+
+        var inputHTML = `<input type="text" value="${potentialAnswer}" class="q-input">`; //pot answer is either '' or what is stored
+        var newTRHtml = '<tr><td>' + question + '</td><td>' + inputHTML + "</td></tr>";
+        
+        if ($('#pitQuestionTable').html().indexOf(question) < 0) //if table doesn't already have it
+        {
+            $('#pitQuestionTable').append(newTRHtml);
+        }
+    }
+}
+
+function savePitAnswers() //onclick for save pit button
+{
+    //using local storage to save pit answers, one localStorage obj per team per event
+    // { '/{season}/{event}/{team}/pit answers' : {{question1}:{answer1}, {question2}:{answer2}...} }
+    //local storage obj will hold an array of q/a object pairs
+
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    var team = getCurrTeamNumber();
+
+    var all_team_QA_pairs_obj = [];
+    $('#pitQuestionTable tr').each(function(){ //for each row
+        var question = $(this).find('td:first').text();
```

Comment:
I wonder if we should put 'question' and 'answer' as classes on the row, so this would become:
`$(this).find('.question').text()`
and
`$(this).find('.answer').val();`